cmd/arcaflow/main.go

```diff
@@ -58,19 +58,22 @@ func main() {
 	dir := "."
 	workflowFile := "workflow.yaml"
 	printVersion := false
+	getNamespaces := false
 
 	const (
-		versionUsage  = "Print Arcaflow Engine version and exit."
-		configUsage   = "The path to the Arcaflow configuration file to load, if any."
-		inputUsage    = "The path to the workflow input file to load, if any."
-		contextUsage  = "The path to the workflow directory to run from."
-		workflowUsage = "The path to the workflow file to load."
+		versionUsage       = "Print Arcaflow Engine version and exit."
+		configUsage        = "The path to the Arcaflow configuration file to load, if any."
+		inputUsage         = "The path to the workflow input file to load, if any."
+		contextUsage       = "The path to the workflow directory to run from."
+		workflowUsage      = "The path to the workflow file to load."
+		getNamespacesUsage = "Show the namespaces available to this workflow."
```

Comment:
This thing of keeping the `=` aligned generates a lot of unnecessary whitespace changes. Yeah, we can disable whitespace diffs, but *sometimes* they're relevant and I don't like to start that way. Goes the Go linter really (stupidly 🤣) insist on this?

Reply:
It's definitely nicer to look at, with the only downside being the diff's readability.

Reply:
Yeah I just accepted this formatting from goland when I started.